packages/components/src/components/input/input.scss

```diff
@@ -227,9 +228,8 @@
         }
     }
     &__hint {
+        margin-bottom: -1.8rem;
```

Comment:
I tried using margin-top it doesn't go the way i wanted. It will move down from the input field which is not the result i wanted. I have to margin-bottom with negative value to minimise the margin from `.dc-input`